### ISSUE TITLE
Catch Error on Pop up close

### ISSUE THREAD

**Issue by farhansaeed27** [alexandrtovmach/react-microsoft-login#18]

**Comment by alexandrtovmach** [comment]
Thank for the issue. Could you please provide a bit more details about it.

**Comment by farhansaeed27** [comment]
msalInstance.loginPopup(scope).then(function (idToken) {
      debug && _this.log("Fetch Azure AD 'token' with popup SUCCEDEED", idToken);
      debug && _this.log("Fetch Graph API 'access_token' in silent mode STARTED", true);

      _this.getGraphAPITokenAndUser(msalInstance, scope, withUserData, authCallback, false, debug);
    });
No Catch...Because it throws Error if user closes popup with signin

**Comment by farhansaeed27** [comment]
```
msalInstance.loginPopup(scope).then(function (idToken) {
      debug && _this.log("Fetch Azure AD 'token' with popup SUCCEDEED", idToken);
      debug && _this.log("Fetch Graph API 'access_token' in silent mode STARTED", true);

      _this.getGraphAPITokenAndUser(msalInstance, scope, withUserData, authCallback, false, debug);
    });
```
No Catch Handling...Because It throws error if user close popup without signin.

**Comment by alexandrtovmach** [comment]
Thanks for issue and details. Fixed by #23 and available in [1.4.2 version](https://www.npmjs.com/package/react-microsoft-login/v/1.4.2)
